tcms/rpc/api/testcase.py

```diff
@@ -304,9 +338,19 @@ def filter(query=None):  # pylint: disable=redefined-builtin
             "default_tester__username",
             "reviewer",
             "reviewer__username",
+            "setup_duration",
+            "testing_duration",
+            "expected_duration",
         )
         .distinct()
     )
+    return [
+        stringify_values(
+            testcase_dict,
+            keys=["setup_duration", "testing_duration", "expected_duration"],
+        )
+        for testcase_dict in qs.iterator()
+    ]
 
```

Comment:
Here I loop through the list of dicts (representing testcases) and make sure that if the value of the duration keys are not null, they are converted from duration objects to strings as xmlrpc cannot marshal duration objects.